Dumping_script.gm9

```diff
@@ -0,0 +1,309 @@
+# author: zeta , jisagi & Glazed_Belmont 
+
+@start
+set DISPLAY_MODE "PKMN Randomizer script\nTYPE: ______"
+
+labelsel -o -s "Is your game a cartridge or digital?" TYPE_*
+goto start
+
+###CARTRIDGE###
+@TYPE_Cartridge
+
+set DISPLAY_MODE "PKMN Randomizer script\nTYPE_CARTRIDGE 
+ask "Script will begin dumping."
+if not find -o -s C:/*.trim.3ds CARTRIDGE
+	echo "No 3DS cart was detected"
+	goto end
+end
+
+imgmount $[CARTRIDGE]
+mkdir 0:/gm9/out/romfs
+mkdir 0:/gm9/out/exefs
+
+if not cp -o -s G:/content0.game/romfs/ 0:/gm9/out/romfs
+	echo "Dumping romfs failed"
+	goto end
+end
+
+if not cp -o -s G:/content0.game/exefs/ 0:/gm9/out/exefs
+	echo "Dumping exefs failed"
+	goto end
+end
+
+labelsel -o -s "What game are you dumping?" CARTRIDGE_*
+
+##PKMN X##
+@CARTRIDGE_X
+
+set GAMEID 00055D00
+goto Update_Dump
+
+
+##PKMN Y##
+@CARTRIDGE_Y
+
+set GAMEID 00055E00
+goto Update_Dump
+
+
+##PKMN OR##
+@CARTRIDGE_Omega_Ruby
+
+set GAMEID 0011C400
+goto Update_Dump
+
+
+##PKMN AS##
+@CARTRIDGE_Alpha_Sapphire
+
+set GAMEID 0011C500
+goto Update_Dump
+
+
+##PKMN SUN##
+@CARTRIDGE_Sun
+
+set GAMEID 00164800
+goto Update_Dump
+
+
+##PKMN MOON##
+@CARTRIDGE_Moon
+
+set GAMEID 00175E00
+goto Update_Dump
+
+
+##PKMN US##
+@CARTRIDGE_Ultra_Sun
+
+set GAMEID 001B5000
+goto Update_Dump
+
+
+##PKMN UM##
+@CARTRIDGE_Ultra_Moon
+
+set GAMEID 001B5100
+goto Update_Dump
+
+
+#-------------------------------------------------------------------------------#
+
+
+###DIGITAL###
+@TYPE_Digital
+
+set DISPLAY_MODE "PKMN Randomizer script\nTYPE_DIGITAL
+ask "Script will begin dumping."
+
+labelsel -o -s "What game are you dumping?" DIGITAL_*
+
+###PKMN X###
+@DIGITAL_X
+
+set X "A:/title/00040000/00055D00"
+if not isdir -o -s $[X]
+	echo "The game wasn't found.\nMake sure you have Pokemon X installed."
+	goto start
+end
+
+if not ask -o -s "The game was found, Continue?"

```

Comment:
Not sure if these are necessary in the first place, and if they are dont copy paste them but instead put them to digital_dump